settings.py

```diff
@@ -0,0 +1,32 @@
+#########################################
+#										#
+#										#
+#			Global Variables			#
+#										#
+#										#
+#########################################
+
+
+PREFIX = "::"
+TOKEN = "NTYzODQ4MDcxMjE0MjY4NDI5.XMjofQ.c0Chs2b0q4Wjp2yqh3MuWfhDU0M"
+AUTHOR_ID=289426079544901633
+
+#can be both str and discord snowflake IDs.
+POLL_ALLOWED_CHANNELS = ["polls"]
+
+#emojis dict. May be possible to change incomprehensible unicode to other strings recognized by discord
+EMOJIS = {
+	"thumbsup": "\U0001f44d",
+	"thumbsdown": "\U0001f44e",
+	"shrug": "\U0001f937",
+}
+
+ADMIN_ROLE = ["Server Admin", "Bot Admin"]
+GESTION_ROLES = ["Community Manager", "Server Admin"]
+DEV_ROLES = ["Dev"]
+for role in ADMIN_ROLE:
+	GESTION_ROLES.append(role)
+	DEV_ROLES.append(role)
+
+SLAPPED_LOG_FILE = "slapped.txt"
```

Comment:
I think it would be better to put constants like that in concerned files (slapping.py for thi by exemple), it will reduce problems with merge and if we want "everyone" to be able to add cogs I think it's better

Reply:
I don't really see why. To me it's a nice way to be know immediately where the info is and have a nice way to change it. Particularly useful when #4 will be done. Now I stay open to your arguments and I may change my mind ;)

Reply:
In fact it's because by this way if someone just want to add the code, or re use only one part (like slapping without todo) he'll have some code for nothing. But it's also easier and less restrictive if someone want to enhance the bot and add a new part.

Reply:
never saw it that way but it's true. We should talk about it tomorrow. Both ideas have potential we just need to determine with precision the objectives of this bot.